src/pages/DMSPolicyInfo.css

```
@@ -49,19 +59,43 @@
 .subtitle-section .section-text {
   padding-top: 10px;
 }
+/*
+  This means that when the window is less than 80px wide (e.g. grid breakpoint = l) then the laptop image will be hidden
+*/
+@media (max-width: 800px) { 
+  .subtitle-section {
+    margin-top: 0;
+    
+  }
+  .subtitle-section img {
+    display: none;
+  }
+}
 
 .store-data-section {
   background-color: #ececec; /* d8d8d8d with 0.5 opacity over white */
   padding: 50px 0;
 }
 .store-data-section img {
   width: 100%;
-  padding-left: 40px;
 }
-
+/* remove padding in mobile mode */
```

```suggestion
/* add padding in desktop view */
```

I would either put `/* no padding in mobile view */` on the `.store-data-section` without media queries, or I would recommend using my suggestion instead. As is it seems a bit confusing to have the comment above the media query. Ditto for the other cases.

---

ah yeah, that's a fair point.  Will update